Assets/LeapMotionModules/Hands/Scripts/LeapHandsAutoRig.cs

```diff
@@ -0,0 +1,302 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Leap.Unity {
+  /**LeapHandAutoRig automates setting up the scripts that drive 3D skinned mesh hands. */
+  [ExecuteInEditMode]
+  [AddComponentMenu("Leap/Auto Rig Hands")]
+  public class LeapHandsAutoRig : MonoBehaviour {
+    public HandPool HandPoolToPopulate;
+    public Animator AnimatorForMapping;
+
+    public string ModelGroupName = null;
+    [Tooltip("Set to True if each finger has an extra trasform between palm and base of the finger.")] 
+    public bool UseMetaCarpals;
```

Comment:
You may want to consider making many/all of these fields non-public.  Since you rely on validation for a lot of the work, and validation is not triggered if the values are changed via script.  It might be a good idea to prevent modification from script altogether to prevent entering an invalid state.